apidoc/Titanium/UI/TabGroup.yml

```diff
@@ -482,6 +485,24 @@ properties:
     default: true
     platforms: [iphone, ipad]
 
+  - name: activeTintColor
+    summary: The activeTintColor to apply to tabs.
+    description: |
+        This property affects the tint of selected tab icons.
+    type: String
+    since: "9.1.0"
+    default: null
+    platforms: [iphone, ipad, android]
+
+  - name: tintColor
```

Comment:
In iOS , New introduced TabGroup properties 'activeTintColor' and 'tintColor' is not behaving same to corresponding properties 'tabsTintColor' and 'unselectedItemTintColor'. See following test case- 
`const tabGroup = Ti.UI.createTabGroup({
  // tintColor: 'green',
  // activeTintColor: 'red'

  unselectedItemTintColor: 'green',
  tabsTintColor: 'red',
});

const tabWin_home = Ti.UI.createWindow({
    title: 'Home',
    backgroundColor: 'white'
});
const tab_home = Ti.UI.createTab({
    icon: 'tab1.png',
    title: 'Home',
    window: tabWin_home
});

const tabWin_chat = Ti.UI.createWindow({
  title: 'Chat',
  backgroundColor: 'white'
});
const tab_chat = Ti.UI.createTab({
  icon: 'tab2.png',
  title: 'Chat',
  window: tabWin_chat
});

tabGroup.addTab(tab_home);
tabGroup.addTab(tab_chat);
tabGroup.open();`

Probably you have to implement these properties similar to [tabsTintColor](https://github.com/appcelerator/titanium_mobile/blob/b890f7c7743b8e38292c5d669ed502996b665a20/iphone/Classes/TiUITabGroup.m#L368) and [unselectedItemTintColor](https://github.com/appcelerator/titanium_mobile/blob/b890f7c7743b8e38292c5d669ed502996b665a20/iphone/Classes/TiUITabGroup.m#L373) .